### PR TITLE
feat: support `npx playwright install msedge`

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

File: bin/reinstall_msedge_stable_mac.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+set -x
+
+cd /tmp
+curl -o ./msedge_stable.pkg -k "$1"
+# Note: there's no way to uninstall previously installed MSEdge.
+# However, running PKG again seems to update installation.
+sudo installer -pkg /tmp/msedge_stable.pkg -target /
```
Comment (yury-s) [inline bin/reinstall_msedge_stable_mac.sh:9]: delete .pkg after installation?

**Review by yury-s — APPROVED** [review]

File: bin/reinstall_msedge_stable_mac.sh (hunk 1 above)
Comment (aslushnikov) [reply bin/reinstall_msedge_stable_mac.sh:9]: done

**Review by yury-s — APPROVED** [review]

File: bin/reinstall_chrome_stable_mac.sh [hunk 2]
```diff
@@ -4,7 +4,9 @@ set -x
 
 rm -rf "/Applications/Google Chrome.app"
 cd /tmp
-curl -o ./googlechrome.dmg -k https://dl.google.com/chrome/mac/beta/googlechrome.dmg
+curl -o ./googlechrome.dmg -k https://dl.google.com/chrome/mac/stable/GGRO/googlechrome.dmg
```
Comment (yury-s) [inline bin/reinstall_chrome_stable_mac.sh:7]: just curious what's `stable/GGRO`

**Review by yury-s — APPROVED** [review]

File: bin/reinstall_chrome_stable_mac.sh (hunk 2 above)
Comment (aslushnikov) [reply bin/reinstall_chrome_stable_mac.sh:7]: 🤷‍♂️

**Pull request merged**